docs/api/session.md

```diff
@@ -50,11 +50,11 @@ const session = require('electron').session;
 const ses = session.fromPartition('persist:name');
 ```
 
-### Instance Events
+## Events
```

Comment:
Trying to remember, is reducing these an issue? Is this related to what was discussed at https://github.com/electron/electron/pull/5168#issuecomment-210460464?